### PR TITLE
Convert menu actions to use umb-button-ellipsis component

### PR DESCRIPTION
**Pull request by BatJan**
### Prerequisites

- [x] I have added steps to test this contribution in the description below

### Description
@nul800sebastiaan this PR converts the old markup pattern used for menu actions to using the umb-button-ellipsis component as you mentioned in #8132 😃

### CODE REVIEW

**Comment by BatJan** [comment]
@nul800sebastiaan After the merge of #8500 I have updated this PR to make use of "mode" as well - I hope the merge went well. Looks fine on my machine so I think it should be OK 😅

File: src/Umbraco.Web.UI.Client/src/views/components/property/property-actions/umb-property-actions.html [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
-        <button
-            class="umb-property-actions__menu-open-toggle"
-            ng-click="vm.close()" tabindex="-1"><i></i><i></i><i></i>
-        </button>
+
+        <umb-button-ellipsis
+            css-class="umb-button-ellipsis--absolute umb-property-actions__menu-open-toggle"
+            action="vm.close()"
+            mode="small"
+            text="Close Property Actions"
```
Comment (nul800sebastiaan) [inline src/Umbraco.Web.UI.Client/src/views/components/property/property-actions/umb-property-actions.html:17]: @BatJan Aren't you missing the `label-key` here? 😁

**Comment by nul800sebastiaan** [comment]
Thanks @BatJan I was about to ask about this! Good stuff! Left you a question!

File: src/Umbraco.Web.UI.Client/src/views/components/property/property-actions/umb-property-actions.html (hunk 1 above)
Comment (BatJan) [reply src/Umbraco.Web.UI.Client/src/views/components/property/property-actions/umb-property-actions.html:17]: Doh! Yes I am :-)

Comment (BatJan) [reply src/Umbraco.Web.UI.Client/src/views/components/property/property-actions/umb-property-actions.html:17]: @nul800sebastiaan  Should be fixed now 👍

**Comment by nul800sebastiaan** [comment]
Thanks @BatJan! The ellipsis are sitting a few pixels lower than before, can you have a look at that too please?

On 8.6.4:
image


On this PR:

image

The gray box is also a little bigger than before it seems.

**Comment by BatJan** [comment]
@nul800sebastiaan Good catch - I'm having a hard time seeing the gray color on my screen so did not notice - It should be better now though 😃

**Comment by nul800sebastiaan** [comment]
Aha! Well let's take the discussion about the lack of proper contrasts some other day then! 😅 

Looks perfectly great now, thanks very much @BatJan! 👍

**Pull request merged**